apps/overige-object-api/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@frameless/overige-object-api",
+  "version": "0.0.0",
+  "private": true,
+  "author": "@frameless",
+  "description": "Overige Object API for PDC",
+  "license": "EUPL-1.2",
+  "keywords": [],
+  "scripts": {
+    "prebuild": "yarn clean && yarn generate-types",
+    "build": "npm-run-all --parallel build:*",
+    "build:server": "tsc -p ./tsconfig.json",
+    "build:openapi": "mkdir -p dist/src/docs && cp -R src/docs/ dist/src/docs/",
+    "watch": "tsc -p ./tsconfig.json  -w",
+    "start": "NODE_ENV=production node ./dist/src/server.js",
+    "dev": "NODE_ENV=development nodemon src/server.ts",
+    "clean": "rimraf dist src/types",
+    "generate-types": "openapi-typescript src/docs/openapi.yaml --output src/types/openapi.ts",
+    "test": "OVERIGE_OBJECT_API_PORT=3000 jest --coverage --forceExit --verbose",
+    "test:watch": "OVERIGE_OBJECT_API_PORT=3000 jest --watch"
+  },
+  "dependencies": {
+    "cors": "2.8.5",
+    "dotenv": "16.4.5",
+    "express": "4.21.0",
+    "express-openapi-validator": "5.3.7",
+    "js-yaml": "4.1.0",
+    "swagger-ui-express": "5.0.1"
+  },
+  "devDependencies": {
+    "@types/cors": "2.8.17",
+    "@types/jest": "29.5.12",
+    "@types/supertest": "6.0.2",
+    "@types/swagger-ui-express": "4.1.6",
+    "@types/yamljs": "0.2.34",
+    "jest": "29.7.0",
+    "nodemon": "3.1.7",
+    "openapi-typescript": "7.4.1",
+    "rimraf": "6.0.1",
+    "supertest": "7.0.0",
+    "ts-jest": "29.2.3",
+    "ts-node": "10.9.2",
+    "typescript": "5.0.4",
+    "jest-fetch-mock": "3.0.3"
+  },
+  "repository": {
+    "type": "git+ssh",
+    "url": "git@github.com:frameless/strapi.git",
+    "directory": "apps/overige-object-api"
+  }
+}
```

Comment:
In de toekomst is het misschien leuk om twee versies te hebben van de Overige Objecten API:

1. Strapi als bron. Optioneel, bijvoorbeeld met `npm run start --strapi https://strapi.localhost:1337/"`
2. Filesystem als bron. Bijvoorbeeld `npm run start --object-dir objects/ --object-types-dir object-types/`

Utility functions om dit te maken zouden ook handig zijn voor tests. Bijvoorbeeld:

```js
let server;
beforeEach(() => {
  server = serverFromFileSystem({
    objectDir: 'fixtures/scenario-5/objects',
    objectDir: 'fixtures/scenario-5/objects-types',
  });
});
```